internal/file/finder.go

```diff
@@ -108,6 +113,38 @@ func (finder *Finder) GetGroups(options DebrickedOptions) (Groups, error) {
 			return nil
 		},
 	)
+	noGroupsFound = len(groups.groups) == 0
+	if noGroupsFound {
+		// Traverse again to see if manifest and lock files were excluded.
+		var excludedGroups Groups
+		err = filepath.Walk(
+			options.RootPath,
+			func(path string, fileInfo os.FileInfo, err error) error {
+				if err != nil {
+					return err
+				}
+				if !fileInfo.IsDir() {
```

Comment:
to reduce the complexity to read this code, you could check, if dir, then return nil directly, instead of nestling the for-loop in the if-statement